OutboundCallReminder/src/main/java/com/communication/outboundcallreminder/OutboundCallReminder.java

```diff
@@ -356,6 +356,15 @@ private void registerToAddParticipantsResultEvent(String operationContext) {
             OperationStatus operationStatus = addParticipantsUpdatedEvent.getStatus();
             if (operationStatus.equals(OperationStatus.COMPLETED)) {
                 Logger.logMessage(Logger.MessageType.INFORMATION, "Add participant status -- > " + operationStatus);
+                
+                //Remove this sleep logic once the call disconnection issue when two PSTN participants present in the call and app exited
```

Comment:
let us remove this comment. our documentation's known issues will already address this issue